### DISCUSSION TITLE
string templates

### DISCUSSION THREAD

**Discussion by artydev** [grucloud/bau#99] [Ideas]
I know it goes against Bau philosophy, but would it be posssible
to allow the html template strings ?
Thank you

**Comment by FredericHeem** [comment]
Allowing 2 different ways to build the UI (functions and template string) would make it more complex, since i am fighting against complexity, I will not bother adding template strings. Moreover, I cannot see any benefits of template string over functions.
Simplicity is the ultimate sophistication.
